news/models.py

```diff
@@ -7,3 +7,71 @@ class Subscriber(models.Model):
 
     def __str__(self):
         return self.user.username
+
+
+class Agency(models.Model):
+    user = models.OneToOneField(get_user_model(), on_delete=models.CASCADE)
+    # todo add other fields
+
+    def __str__(self):
+        return self.user.username
+
```

Comment:
This model name doesn't seem correct. Also Isn't it better to have a OneToOneField to Subscriber?